src/compas_blender/artists/networkartist.py

```diff
@@ -92,7 +116,7 @@ def object_edge(self, values):
         self._object_edge = dict(values)
 
     @property
-    def object_path(self):
+    def object_path(self) -> Dict[bpy.types.Object, List[int]]:  # !!! what is a path?
```

Comment:
a polyline, i think, so a list of points, but not sure :)
perhaps just remove entirely because there is no matching implementation...